posts/2022-09-22-real-time-everything.md

```diff
@@ -0,0 +1,249 @@
+---
+title: 'Real-time “everything” with operational and analytics data'
+description: 'Breakdown the walls between operational and analytics for real-time data sharing, starting with a shared love language to translate between JSON and SQL'
+date: '2022-09-22'
+categories:

```

Comment:
Curious if these categories "exist" to the blog.  Could be why the build is failing - https://github.com/vendia/site/actions/runs/3094563903/jobs/5008074687#step:5:2813?

Reply:
They seem to all be listed in the categories.